src/extensions/custom-view/bootstrap-table-custom-view.js

```diff
@@ -12,7 +12,16 @@ $.extend($.fn.bootstrapTable.defaults, {
 })
 
 $.extend($.fn.bootstrapTable.defaults.icons, {
-  customView: {
+  customViewOn: {
```

Comment:
I'm fine if you want to change the "hide custom view"-Button.
We already have a little BC with the renaming, which is fine so we also can improve the UX by adding/using a other icon.

Reply:
I would suggest to use below icons then:

```
$.extend($.fn.bootstrapTable.defaults.icons, {
  customViewOn: {
    bootstrap3: 'glyphicon glyphicon-list',
    bootstrap5: 'bi-list',
    bootstrap4: 'fa fa-list',
    semantic: 'fa fa-list',
    foundation: 'fa fa-list',
    bulma: 'fa fa-list',
    materialize: 'list'
  }[$.fn.bootstrapTable.theme] || 'fa-list',
  customViewOff: {
    bootstrap3: 'glyphicon glyphicon-thumbnails',
    bootstrap5: 'bi-grid',
    bootstrap4: 'fa fa-th',
    semantic: 'fa fa-th',
    foundation: 'fa fa-th',
    bulma: 'fa fa-th',
    materialize: 'grid_on'
  }[$.fn.bootstrapTable.theme] || 'fa-th'
})
```

(assuming fontawesome 5 is used)

@UtechtDustin If you are fine with this proposal then let me know and I will commit that change.

Reply:
Looks good